capture/src/main/kotlin/com/alfresco/capture/CameraFragment.kt

```diff
@@ -176,7 +182,7 @@ class CameraFragment : Fragment(), KeyHandler, MavericksView {
         layout.viewFinder.controller = cameraController
 
         // Observe zoom changes
-        cameraController?.zoomState?.observe(this) {
+        cameraController?.zoomState?.observe(viewLifecycleOwner) {
```

Comment:
This change has nothing to do with this PR.

Reply:
Ok. I will undo this